src/world.rs

```diff
@@ -305,6 +305,21 @@ impl World {
         })
     }
 
+    /// Given the ID of an entity without a generation, try to find the matching generation.
+    ///
+    /// This function is useful if you know there's an element with a given index, but you
+    /// don't know its generation.
+    ///
+    /// # Safety
+    /// This function can still return `Some` if passed the ID of a dead entity. In this case,
+    /// it will return an entity which will, if looked up, have no components, but which may
+    /// later become allocated with some other set of components. Only use this if you're
+    /// absolutely certain that the entity is live, or if you have some other way to ensure
+    /// you have acquired a live entity!
```

Comment:
```suggestion
    /// `id` must correspond to a currently live `Entity`. A despawned or never-allocated `id` will produce undefined behavior.
```

We do not want to guarantee any particular behavior here.